action.yml

```diff
@@ -1,22 +1,34 @@
 name: 'NGINX configuration sync'
-description: 'The action synchronizes NGINX configuration from a Git repository to an NGINX deployment on Azure.'
+description: 'The action synchronizes NGINX configuration from a Git repository to an NGINX for Azure deployment.'
 inputs:
   subscription-id:
-    description: 'The Azure subscription ID of the NGINX deployment'
+    description: 'The Azure subscription ID of the NGINX for Azure deployment.'
     required: true
   resource-group-name:
-    description: 'The resource group of the NGINX deployment'
+    description: 'The resource group of the NGINX for Azure deployment.'
     required: true
   nginx-deployment-name:
-    description: 'The name of the NGINX deployment'
+    description: 'The name of the NGINX for Azure deployment.'
     required: true
-  nginx-config-relative-file-path:
-    description: 'The relative file path of the NGINX configuration file in the Git repository'
+  nginx-config-directory-path:
+    description: 'The NGINX configuration directory path relative to the root of the Git repository, example: "config".'
     required: true
-    default: './config/nginx.conf'
+  nginx-root-config-file-name:
+    description: >
+     'The root NGINX configuration file name, example: "nginx.conf". The root file must be in the NGINX configuration directory 
+     but not in its subdirectory.'
```

Comment:
I'm having a hard time understanding what this means. It's not clear what "its subdirectory" refers to.

Reply:
Removed this requirement. Renamed the parameter to nginx-root-config-file and the root file's relative path to the config directory can be specified.